.vscode/launch.json

```diff
@@ -34,16 +34,7 @@
 			"outFiles": [
 				"${workspaceFolder}/packages/foam-vscode/out/test/**/*.js"
 			],
-			"preLaunchTask": "Build foam-vscode"
-		},
-		{
-			"type": "node",
-			"request": "launch",
-			"name": "Workspace Manager tests",
```

Comment:
Keeping some kind of entry point to debug the core tests would be nice? We could put this also into foam-core launch.json, but it would probably be somewhat annoying to need to load that folder into separate vscode instance. Thus I'd still keep this here (and name it e.g. to "foam-core unit tests" or something similar.

Reply:
Ah yes, I didn't think of debugging tests, I was making the assumption that we would just run tests from the command line but that is clearly insufficient!